drivers/scheduler/k8s/specs/wordpress/wordpress.yaml

```diff
@@ -48,7 +46,46 @@ spec:
         volumeMounts:
         - name: wordpress-persistent-storage
           mountPath: /var/www/html
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 80
+          initialDelaySeconds: 15
+          timeoutSeconds: 5
+        resources:
+          requests:
+            memory: "2Gi"
```

Comment:
is this a must? I would refrain from requesting memory since this will run in a test context and will contend with other pods for memory.

Reply:
This is our customer's recommended size for woocommerce sites. With lower values we were regularly seeing seige push the memory use high enough for Kubernetes to kill the pods regularly.